### PR TITLE
Update docs as Kairos AMIs are no longer supported

### PR DESCRIPTION
**Pull request by 2dvorak**
## Proposed changes

- We stop providing AMIs for Kairos starting from November 1, 2024.
- Update docs accordingly

## Types of changes

Please put an x in the boxes related to your change.

- [x] Minor Issues and Typos
- [ ] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/kaiachain/kaia-docs/blob/main/CONTRIBUTING.md)
- [x] I have read the [CLA](https://gist.github.com/kaiachain-dev/bbf65cc330275c057463c4c94ce787a6) and signed by comment ```I have read the CLA Document and I hereby sign the CLA``` in first time contribute
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/misc/operation/aws-ami.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 Kaia offers an AWS AMI ([Amazon Machine Image](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html)) for Kaia endpoint nodes (EN). These AMIs are pre-installed with both the EN software and the chaindata, enabling users to quickly and conveniently launch a fully operational node. With Kaia's AMI, setting up a new endpoint node on demand becomes seamless, allowing for simplifying the process of joining the Kaia network.
 
+> Note: The Kaia Foundation provided AWS AMIs for the Kairos testnet until October 29, 2024, and will cease to provide these after November 1, 2024
```
Comment (blukat29) [inline docs/misc/operation/aws-ami.md:5]: Do we need to mention twice? the one below the table seems enough.

File: docs/misc/operation/aws-ami.md [hunk 2]
```diff
@@ -29,7 +33,7 @@ We generate the latest AMI every day around 6 pm KST. Once generated, the AMIs l
 
 ## Launch new EC2 instance with AMI in Amazon console
 
-When launching a new EC2 instace in the AWS console, you should select an AMI. Search for `kaia-mainnet` or `kaia-kairos` in the AMI search bar.
+When launching a new EC2 instace in the AWS console, you should select an AMI. Search for `kaia-mainnet` in the AMI search bar.
```
Comment (ayo-klaytn) [inline docs/misc/operation/aws-ami.md:36]: small typo : instace to instance.

File: docs/misc/operation/aws-ami.md (hunk 1 above)
Comment (2dvorak) [reply docs/misc/operation/aws-ami.md]: Tried to make sure readers do not miss it.
Updated!

File: docs/misc/operation/aws-ami.md (hunk 2 above)
Comment (2dvorak) [reply docs/misc/operation/aws-ami.md]: Done, thanks!

**Review by blukat29 — APPROVED** [review]

File: docs/misc/operation/aws-ami.md (hunk 2 above)
Comment (2dvorak) [reply docs/misc/operation/aws-ami.md]: @ayo-klaytn Please take a look and approve this PR if you don't find any other issues.

**Review by scott-klaytn — APPROVED** [review]

**Pull request merged**